src/main/java/com/ericsson/ei/controller/AuthControllerImpl.java

```diff
@@ -34,12 +34,12 @@
 import io.swagger.annotations.ApiOperation;
 
 /**
- * Endpoints /auth/login and /auth/checkStatus should be secured in case LDAP is enabled
- * Endpoint /auth should be not secured
+ * Endpoints /auth/login should be secured in case LDAP is enabled Endpoint
+ * /auth should be not secured
```

Comment:
be not --> not be. Also add a full stop between the sentences.

Reply:
Done.